api/v1alpha1/release_types.go

```diff
@@ -60,6 +60,11 @@ func (rr ReleaseReason) String() string {
 	return string(rr)
 }
 
+const (
+	// AutoReleaseLabel is the label name for the auto-release setting
+	AutoReleaseLabel = "release.appstudio.openshift.io/auto-release"
+)
+
 // ReleaseStatus defines the observed state of Release
```

Comment:
Any reason this one does not have the "."?

Reply:
No. I'll fix it.